### PR TITLE
Initial attempt at deploying to gradle plugin portal

### PR DESCRIPTION
**Pull request by jimschubert**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`~~, `3.4.x`, `4.0.x`~~. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

This is an attempt at publishing the gradle plugin to the gradle plugin portal.
I've attempted a publish locally, but it is pending approval: https://plugins.gradle.org/plugin/org.openapi.generator

### CODE REVIEW

File: modules/openapi-generator-gradle-plugin/build.gradle [hunk 1]
```diff
@@ -90,6 +96,23 @@ gradlePlugin {
...
+    website = 'https://openapi-generator.tech/'
+    vcsUrl = 'https://github.com/OpenAPITools/openapi-generator'
+    description = 'OpenAPI Generator allows generation of API client libraries (SDK generation), server stubs, documentation and configuration automatically given an OpenAPI Spec (v2, v3)'
+
+    plugins {
+        // first plugin
+        openApiGenerator {
+            id = 'org.openapi.generator'
+            displayName = 'OpenAPI Generator Gradle Plugin'
+            tags = ['openapi-3.0', 'openapi-2.0', 'openapi', 'swagger', 'codegen', 'sdk']
```
Comment (jimschubert) [inline modules/openapi-generator-gradle-plugin/build.gradle]: I wasn't sure what tags to use here.

Comment (wing328) [reply modules/openapi-generator-gradle-plugin/build.gradle]: I think those are good to start with and we always revise and add more later based on the feedback from the community.

**Comment by wing328** [comment]
Looks good. Let's try it.

**Comment by jimschubert** [comment]
@wing328 I just received an email that the plugin was approved: https://plugins.gradle.org/plugin/org.openapi.generator

I think this should be merged tomorrow so we can compare the publish dates. The first published plugin is March 19, 2019

**Pull request merged**